apps/judicial-system/web/src/routes/Shared/Cases/ActiveCases.tsx

```diff
@@ -267,13 +271,18 @@ const ActiveCases: React.FC<Props> = (props) => {
                       <Box component="span" className={styles.blockColumn}>
                         <Text as="span">{c.courtCaseNumber}</Text>
                       </Box>
-                      <Text as="span" variant="small" color="dark400">
-                        {c.policeCaseNumbers.join(', ')}
+                      <Text
+                        as="span"
+                        variant="small"
+                        color="dark400"
+                        title={c.policeCaseNumbers.join(', ')}
```

Comment:
Title prop ensures that you can hover over the element to see all Police Case numbers.